doxygen/dox/IntroHDF5.dox

```diff
@@ -608,7 +608,6 @@ on the <a href="http://hdfeos.org/">HDF-EOS Tools and Information Center</a> pag
 \section secHDF5Examples Examples
 \li \ref LBExamples
 \li \ref ExAPI
-\li <a href="https://github.com/HDFGroup/hdf5/tree/develop/examples">Examples in the Source Code</a>
```

Comment:
Shouldn't this be pointing to the HDF5Examples source: https://github.com/HDFGroup/hdf5/blob/develop/HDF5Examples